tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -234,11 +234,23 @@ Wait Until JupyterLab Is Loaded
   Wait Until Element Is Visible  xpath:${JL_TABBAR_CONTENT_XPATH}  timeout=${timeout}
 
 Clone Git Repository
```

Comment:
There was already `Clone Git Repository` and `Clone Git Repository And Open` and this RP is adding two new keywords, `Clone Repo` and `Clone Repo and Return Error Message`.  I think it will add more confusion. 

Is it possible just modify `Clone Git Repository`  for the new needs?
- Add a parameter ${delete-existing-clone}  (enabled by default) the will preventively delete the folder if it exists already
- Log error messages

Reply:
If we just modify the `Git Clone Repository` keyword without having keywords like `Clone Repo` and 'Clone Repo and Return Error Message' then it will not look good.